crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -349,7 +356,7 @@ impl SweetConductor {
     /// better to do that once for all apps in the case of multiple apps being
     /// set up at once.
     #[cfg_attr(feature = "instrument", tracing::instrument(skip_all))]
-    async fn setup_app_3_create_sweet_app(
+    async fn create_sweet_app_for_installed_app(
```

Comment:
I found the `setup_app_x_...` function names a bit confusing personally so I renamed it to a name that seemed more descriptive to me.